DOCS.md

```diff
@@ -69,270 +70,325 @@ In a Sponsored Ad Break flow, once the user hits a mid-roll break with a true[X]
 
-If there were no ads available to the user or if there was an error making the ad request, `NO_ADS_AVAILABLE` or `AD_ERROR` will fire respectively. At this point, the app should resume playback without skipping any ads, so the user receives a normal video ad payload.
+### Handling Ad Elimination
 
-Another important event to listen for is `AD_FREE_POD`. This signifies that the user has earned a credit with true[X] and, once the renderer signals it is complete, all linear video ads remaining in the current pod should be skipped. It's important to note that the player should not immediately resume playback once receiving this event -- rather it should note that it was fired and continue to wait for an `AD_COMPLETED` event.
+Skipping video ads is completely the responsibility of the app code. The SSAI API should provide enough information for the app to determine where the current pod end-point is, and the app, when appropriate, should fast-forward directly to this point when resuming playback.
 
-The last major event to fire will be an `AD_COMPLETED` event. This will signal to the app that the renderer has finished showing the true[X] unit to the user. The user may have opted out of true[X], let the choice card countdown expire, or may have completed a true[X] ad unit and earned an ad-free credit (an `AD_FREE_POD` event would have already been fired in the last case). At this point, the app should check if an `AD_FREE_POD` event has fired -- if so, the user should be fast-forwarded past all remaining linear video ads and playback should resume. If not, playback still should resume, but without fast-forwarding so the user receives the remaining payload of video ads.
 
+## TruexAdRenderer tvOS API
 
-### Handling Ad Elimination
+This is an outline of public `TruexAdRenderer` methods and `TruexAdRendererDelegate` methods (referred to as events).
 
-Skipping video ads is completely the responsibility of the app code. The Uplynk Preplay API should provide enough information for the app to determine where the current pod end-point is, and, when appropriate, should fast-forward directly to this point when resuming playback.
 
+### Adding `TruexAdRenderer` to Your Project
 
-## TruexAdRenderer tvOS API
+The easiest way to add `TruexAdRenderer` is via CocoaPods. The renderer will be maintained and distributed on Github.
 
-This is an outline of public `TruexAdRenderer` methods and delegate methods (referred to as events).
+1. Add the following Podspec repositories to your Podfile:
+    ```ruby
+    source 'https://github.com/CocoaPods/Specs.git'
+    source 'https://github.com/socialvibe/cocoapod-specs.git'
+    source 'https://innovid-cocoapods:3bbe7445d58951a8f89c5e54c0ebfdd17039589e@github.com/Innovid/cocoapods-spec.git'
```

Comment:
Ooops, my bad here. Realized we should update to the generic GH repo link: 
```ruby
source 'https://github.com/Innovid/cocoapods-spec.git'
```

I assume this is a relic and their spec repo used to be private. Let me verify this change in the ref app and we can update. Grabbed it from their demo: https://github.com/Innovid/tvos-framework-live-demo

Reply:
Just verified!

https://github.com/socialvibe/truex-tvos-google-ad-manager-reference-app/pull/4

Reply:
ty